onecodex/viz/__init__.py

```diff
@@ -82,38 +82,42 @@
 
 
 def onecodex_theme():
+    import altair as alt
+
     onecodex_palette = ["#ffffcc", "#c7e9b4", "#7fcdbb", "#41b6c4", "#2c7fb8", "#264153"]
 
     font_family = "Fira Sans, Helvetica"
 
-    return {
-        "config": {
-            "range": {
-                "heatmap": list(reversed(onecodex_palette)),
-                "category": DEFAULT_PALETTES["ocx_categorical"],
-                "ramp": list(reversed(onecodex_palette)),
-            },
-            "area": {"fill": OCX_DARK_GREEN},
-            "bar": {"fill": OCX_DARK_GREEN},
-            "mark": {"color": OCX_DARK_GREEN},
-            "axis": {
-                "labelFont": font_family,
-                "labelFontSize": 12,
-                "titleFont": font_family,
-                "titleFontSize": 12,
-                "grid": False,
-            },
-            "legend": {
-                "labelFont": font_family,
-                "labelFontSize": 12,
-                "titleFont": font_family,
-                "titleFontSize": 12,
-            },
-            "title": {"font": font_family},
-            "view": {"width": 400, "height": 400, "strokeWidth": 0},
-            "background": "white",
+    return alt.theme.ThemeConfig(
```

Comment:
technically breaking backwards compat. Do we need to change the return type in order to silence the warning or can `alt.theme.ThemeConfig` be called in `configure_onecodex_theme`?

Reply:
Altair's deprecation warning recommended using this new return type, but I just tested with returning a dict and it seems to work (and doesn't raise the warning). Reverted!